packages/enzyme-adapter-react-16.1/src/ReactSixteenOneAdapter.js

```diff
@@ -83,9 +83,16 @@ function toTree(vnode) {
   switch (node.tag) {
     case HostRoot: // 3
       return childrenToTree(node.child);
-    case HostPortal: { // 4
-      return childrenToTree(node.child);
-    }
+    case HostPortal: // 4
+      return {
+        nodeType: 'portal',
+        type: Portal,
+        props: {},
```

Comment:
I wonder if this shouldn't be `props: { children }` - ie, have the actual children provided to createPortal?

also, perhaps include the `containerInfo` as a "prop"?